tools/benchmarking/frame-weight-template.hbs

```diff
@@ -0,0 +1,103 @@
+// This file is part of Substrate.
```

Comment:
Looks like this was copied from Substrate. Was that on purpose?

Reply:
The file was on purpose, but the legalese wasn't. I'll review this and update the header if I can [legally].

Reply:
The file is a template ("Handlebars", haven't investigated) that allows us to customize what the resulting rust code (the weight definitions) looks like. I like the concept, but it looks like it would take some effort to understand how it works (e.g. to make any modifications).

Reply:
https://handlebarsjs.com/

Reply:
Fixed in 5feed2e